### PR TITLE
[TASK] Href with working links

### PR DESCRIPTION
**Pull request by xf-**
- View is not included (no update on view change)
- fixes view indents as well

WIP -> do not merge

### CODE REVIEW

File: lib/router.js [hunk 1]
```diff
@@ -11,25 +11,21 @@ define(['helper'], function (helper) {
...
-      if (currentView) {
-        e.push('v:' + currentView);
-      }
+      e += 'v:' + currentView;
 
       if (currentObject) {
         if ('node' in currentObject) {
-          e.push('n:' + encodeURIComponent(currentObject.node.nodeinfo.node_id));
+          e += ';n:' + encodeURIComponent(currentObject.node.nodeinfo.node_id);
         }
```
Comment (genofire) [inline lib/router.js]: to be sure, a `else if` instatt of a only `if` - both states are not possible at the same time.

File: lib/router.js [hunk 2]
```diff
@@ -129,6 +125,20 @@ define(['helper'], function (helper) {
       return targetSet;
     }
 
+    self.getUrl = function getUrl(data) {
+      var e = '#!';
+
+      if (data.n) {
```
Comment (genofire) [inline lib/router.js:129]: missing `current view`

**Comment by genofire** [comment]
Should the **Switch View** button on the top right side be also a link?

**Comment by SimJoSt** [comment]
Is this similar to https://github.com/plumpudding/hopglass/pull/84#event-948775006 ?
The description here is not enough to make me understand.

File: lib/router.js (hunk 2 above)
Comment (xf-) [reply lib/router.js:129]: is not missing, only map can be linked. link isset and afterwards not updated.

File: lib/router.js (hunk 1 above)
Comment (xf-) [reply lib/router.js]: yes, could be changed

**Comment by genofire** [comment]
@SimJoSt jep, but more beautifull (and everywhere)

**Comment by xf-** [comment]
@genofire 
I don't think we need a link:
- 1 of 3 buttons would have a link afterwards
- Links in lists are more useful
- Buttons have no href

@SimJoSt yes, but i think urls should be handled routing. Much easier to refactor.

**Pull request merged**